telco-hub/configuration/reference-crs/optional/backup-recovery/dataProtectionApplication.yaml

```diff
@@ -0,0 +1,36 @@
+---
+kind: DataProtectionApplication
+metadata:
+  name: hub-backup
+  namespace: open-cluster-management-backup
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+          s3ForcePathStyle: 'true'
+          s3Url: #your S3 end point
+          insecureSkipTLSVerify: "true"
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: #your bucket name
+          prefix: velero
+        provider: aws
+  configuration:
+    restic:
+      enable: true
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - kubevirt
+  snapshotLocations:
+    - velero:
+        config:
+          profile: default #your velero secret profile name
```

Comment:
Is this the secret created above "cloud-credentialscloud-credentials"? If so, please update the name here so that the user does not have to.

Reply:
this is the profile name.
[default] # this is the profile name : default
aws_access_key_id=xxxxxxxxxxxxxxx
aws_secret_access_key=xxxxxxxxxxxxxx